### ISSUE TITLE
As reviewer, I can see new records arrive in my workflow, so I can review them [Review workflow case]

### ISSUE THREAD

**Issue by mietcls** [ugent-library/biblio-backoffice#351]
# Story

As reviewer, I can see new records arrive in my workflow, so I can review them.

TLDR;
- Initial status: Published & In review
- Actions: Reviewer can view newly published records to review them.

### Status
- - [ ] New, unpublished records added by researchers get the visibility status (Biblio status): Published
- - [ ] New, unpublished records added by researchers get the review status: In review

### Workflow
- - [ ] The record shows up in the reviewer's dashboard
- - [ ] Reviewers can filter on new published records in the overview with the status "In review"

### Visibility
- - [ ] The record is visible to involved researchers (? TBD: or only draft for editor itself?) in their publications
- - [ ] The record is visible to involved researchers on their resumé
- - [ ] The record is visible on the grant summary page
- - [ ] The record is visible in Biblio, available to the rest of the world
- - [ ] The record gets a persistent URL (handle) that can be (re)used by professional instances

### Actions
- - [ ] Reviewers can review this record, and based on their assessment take other actions such as:
- - #353
- - #354
- - #355
- - #356

# Additional info

This story is part of epic #347, presented as one of the cases that fit into the review workflow that involves motivating researchers to complete records.

**Comment by mietcls** [comment]
Closing as outdated
